logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -81,7 +83,11 @@ import kotlin.time.Duration.Companion.minutes
 @Suppress("TooManyFunctions")
 internal interface UserRepository {
     suspend fun fetchSelfUser(): Either<CoreFailure, Unit>
-    suspend fun fetchKnownUsers(): Either<CoreFailure, Unit>
+
+    /**
+     * fetches user information for all of users id stored in the DB
+     */
+    suspend fun syncAllOtherUsers(): Either<CoreFailure, Unit>
```

Comment:
Why prefix this method with `sync` instead of `fetch` like the other methods?

Reply:
in my mind what this function does is fetches users data form remove and store it locally which is closer to a sync operation, i fetchUsersInfo

Reply:
Right makes some sense but the other methods named `fetch` also doesn't return anything. Why not open separate refactoring PR if you want to change how we name these methods.